documentation/docs/pmm-upgrade/external_postgres_pmm_upgrade.md

```diff
@@ -0,0 +1,164 @@
+# Migrate external PostgreSQL configuration for PMM 3.2.0+ upgrades
+
+If you're using an external PostgreSQL database with PMM, you will need to update your configuration before upgrading to PMM 3.2.0 or later. This is because PMM 3.2.0 includes Grafana 11.6, which introduces a [regression issue](https://github.com/grafana/grafana/issues/102337): the single `GF_DATABASE_URL` environment variable is no longer sufficient for database configuration.
+
+To upgrade to PMM 3.2.0 successfully, convert your configuration to use individual environment variables instead of `GF_DATABASE_URL`: 
+
+## Before you begin
+
+- Verify you're using an external PostgreSQL database with PMM
+- Ensure you have the connection details for your PostgreSQL database
+- Back up your PMM data before proceeding
+
+## Migration procedure
+
+### 1. Determine your current database URL format
+
+Currently, your external PostgreSQL is likely configured with a connection string like: `GF_DATABASE_URL=postgres://USER:PASSWORD@HOST:PORT/DATABASE_NAME`. 
+
+You'll need to extract these components for the new configuration format.
+
+### 2. Stop your PMM Server
+Depending on your deployment method, use one of the following approaches:
+
+=== "Docker"
+    ```bash
+    docker stop pmm-server
+    docker rm pmm-server
+    ```
+
+=== "Podman"
+    ```bash
+    podman stop pmm-server
+    podman rm pmm-server
+    ```
+
+=== "Docker Compose"
+    ```bash
+    docker-compose down
```

Comment:
Here it might be good to specify to `down` just the pmm-server, since there isn't probably a reason to stop the other services.


@ademidoff what do you think?